### PR TITLE
global-shortcuts: Responses should succeed if nothing wrong happened

### PR DESCRIPTION
**Pull request by aleixpol**
We were always returning 2, which is not correct.

### CODE REVIEW

File: src/global-shortcuts.c [hunk 1]
```diff
@@ -394,6 +395,7 @@ shortcuts_listed_cb (GObject *source_object,
           results = g_variant_ref_sink (g_variant_builder_end (&results_builder));
         }
 
+      response = 0;
```
Comment (jadahl) [inline src/global-shortcuts.c:398]: These tends to be in the impl method call, e.g. https://github.com/flatpak/xdg-desktop-portal/blob/8255fabc37cd6a943c07a0b469e5a1a944aeaecf/data/org.freedesktop.impl.portal.ScreenCast.xml#L250 and https://github.com/flatpak/xdg-desktop-portal/blob/8255fabc37cd6a943c07a0b469e5a1a944aeaecf/data/org.freedesktop.impl.portal.Inhibit.xml#L85.

There hasn't been a release yet, so we can still add the `response` argument where it's missing.

Comment (aleixpol) [reply src/global-shortcuts.c]: Addressed. Added a couple of extra changes of things that cropped up when testing it.

File: data/org.freedesktop.portal.GlobalShortcuts.xml [hunk 2]
```diff
@@ -160,10 +160,23 @@
...
         @session_handle: Object path for the #org.freedesktop.impl.portal.Session object representing the session
-        @options: Unused so far
+        @options: See below
 
         Lists all shortcuts.
 
+        Supported keys in the @options vardict include:
+        <variablelist>
+          <varlistentry>
+            <term>handle_token s</term>
```
Comment (jadahl) [inline data/org.freedesktop.portal.GlobalShortcuts.xml:170]: Shouldn't `handle_token` be added to `BindShortcuts` too?

**Review by jadahl — APPROVED** [review]
lgtm

**Comment by aleixpol** [comment]
Hi @jadahl, thanks for your review. Please note I do not have powers to merge so somebody will have to press the button for the changes to go in.

**Comment by jadahl** [comment]
> Hi @jadahl, thanks for your review. Please note I do not have powers to merge so somebody will have to press the button for the changes to go in.

I don't have any merge rights either.

File: src/global-shortcuts.c [hunk 3]
```diff
@@ -445,8 +446,7 @@ handle_list_shortcuts (XdpDbusGlobalShortcuts *object,
                                                       request->id,
                                                       arg_session_handle,
                                                       NULL,
-                                                      shortcuts_listed_cb,
-                                                      g_object_ref (request));
+                                                      shortcuts_listed_cb, g_object_ref (request));
```
Comment (GeorgesStavracas) [inline src/global-shortcuts.c:450]: Please don't change this

**Review by GeorgesStavracas — CHANGES_REQUESTED** [review]
One comments, LGTM otherwise

File: src/global-shortcuts.c (hunk 3 above)
Comment (aleixpol) [reply src/global-shortcuts.c]: Addressed.

**Review by GeorgesStavracas — APPROVED** [review]

**Pull request merged**